src/main/java/goodgenerator/loader/RecipeLoader_02.java

```diff
@@ -566,10 +736,34 @@ public static void RecipeLoad() {
 
         GT_Values.RA.addUnboxingRecipe(
                 ItemRefer.YOTTank_Cell_T6.get(1),
-                ItemRefer.Fluid_Storage_Core_T6.get(10),
+                ItemRefer.Fluid_Storage_Core_T6.get(2),
```

Comment:
the recursive recipe should be kept so people can recycle and upgrade the existed tanks